style.css

```diff
@@ -427,3 +438,221 @@ textarea::placeholder {
   margin: 0 auto;
   width: 8rem;
 }
+
+@media only screen and (min-width: 768px) {
+  .mobile-menu {
+    display: none;
+  }
+
+  #navigation {
+    display: flex;
+    justify-content: space-between;
+    padding: 16px 66px;
+    box-shadow: 0 1px 10px -5px grey;
+    position: fixed;
+    height: 72px;
+  }
+
+  nav {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .desktop-links {
+    display: flex;
+    justify-content: space-around;
+    width: 300px;
+    align-items: center;
+  }
+
+  .desktop-links li {
+    list-style: none;
+    color: #344563;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    align-items: center;
+    cursor: pointer;
+  }
+
+  .card-tools li {
+    padding: 4px 4px;
+  }
+
+  .language-list li {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .desktop-links li:hover {
+    color: #6070ff;
+  }
+
+  .hero-section {
+    background-image: url(./images/Header\ bg.svg);
+    background-size: 100% 691.46px;
+    background-position-x: right;
+    padding-top: 8rem;
+    padding-bottom: 9rem;
+    height: 846px;
+  }
+
+  .hero-container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: flex-start;
+    width: 70%;
+    margin: auto;
+    margin-top: 100px;
+    gap: 50px;
+  }
+
+  #name {
+    font-size: 48px;
+    line-height: 60px;
+    letter-spacing: 0.37px;
+    margin: 10px 0;
+    height: 50px;
+  }
+
+  .details {
+    width: 100%;
+    margin-top: 1rem;
+    height: 28px;
+  }
+
+  .connect {
+    transform: translateY(-2rem);
+    margin-top: 1.5rem;
+    height: 0;
+  }
+
+  .hero-section ul {
+    margin-top: -3em;
+  }
+
+  #work-section {
+    display: none;
+  }
+
+  #work-section-desktop {
+    width: 80%;
+    margin: auto;
+    display: grid;
+    grid-template-columns: 1fr;
+    grid-auto-flow: row;
+    justify-items: center;
+    align-items: center;
+    margin-top: 46px;
+  }
+
+  .card {
+    flex-direction: row;
+    justify-content: space-between;
+    gap: 36px;
+    max-width: initial;
+  }
+
+  .card:hover {
+    cursor: pointer;
+    border: 2px solid #a7aeff;
+  }
+
+  .two {
+    flex-direction: row-reverse;
+  }
+
+  .card-img {
+    display: grid;
+    place-items: center;
+    width: 100%;
+  }
+
+  .card-img img {
+    width: 100%;
+  }
+
+  .card-description {
+    margin-right: 1rem;
+  }
+
+  .card-details {
+    height: auto;
+  }
+
+  .card-tools {
+    width: 108%;
+  }
```

Comment:
- [ ] You have done a good job creating the elements inside each card :+1:. However, to ensure that your project adheres to the Figma template, it is necessary to follow the same margins and padding as specified in the template. Currently, there are differences in the spacing between elements, which can affect the overall aesthetic of your project.

**Your design**

![Screenshot from 2023-05-05 04-45-30](https://user-images.githubusercontent.com/105117832/236427167-b73d84c1-7072-4d59-9cca-56e0209b8982.png)

**Figma**

![Screenshot from 2023-05-05 04-45-46](https://user-images.githubusercontent.com/105117832/236427207-810255ce-b98e-4fa2-a3e1-1b726740ef75.png)